### ISSUE TITLE
Corrupted Apk

### ISSUE THREAD

**Issue by Cosciad** [Ahsan40/GLTools#4]
![Screenshot_20200427-163627](https://user-images.githubusercontent.com/37622907/80384802-a0624e80-88a5-11ea-832c-7b6988fa6c5c.png)

Can't install the apk...
Is it possible for you to give only the module somewhere pls ?
Thanks

**Comment by Ahsan40** [comment]
Did you tried without sai? I ment with official installer?

I wanted to give the module but every phone generate different files for module that's why it's not possible. 
But if you want you can follow this way,
1. Download master from this repo & extract is. (it's a empty template)
2. Install official gltools.
3. generate magisk module 
4. extract system folder from that module and place it in downloaded template.
5. zip it and install in magisk.

This should work.....

**Comment by Cosciad** [comment]
Yes I tried without sai, it tells me that the APK is corrupted too =)

**Comment by Ahsan40** [comment]
Then try to create module for your phone with this empty template. 

Anyway I am using custom rom and apk verification is disabled in this one. So  I actually couldn't test any build for the issue you showed. Looks like I have to disable signature verification in apk. It may take lot of time. So I will do it in future when I get time.

**Comment by Cosciad** [comment]
Ok, done and yes it works =)

**Comment by Cosciad** [comment]
Another thing, do you know if it's possible to decrease the size of the fps counter pls ?

**Comment by Ahsan40** [comment]
> 
> 
> Another thing, do you know if it's possible to decrease the size of the fps counter pls ?

I know the way(need to modify size from xml file inside resource) but again if you wanna do that you must need to reverse engineer the apk and disable the signature verification otherwise it will say "Hi Hacker....."

**Comment by Cosciad** [comment]
OK thks =)

**Comment by Infector53** [comment]
It doesn't have a file apk GLTools v4.02 r3.0

**Comment by Infector53** [comment]
doesn't work

**Comment by Infector53** [comment]
https://yadi.sk/d/89GqOj8uFNMYDw

**Comment by Ahsan40** [comment]
> 
> 
> https://yadi.sk/d/89GqOj8uFNMYDw
> with a cut-out signature

I tested this one. Signature verification has been cut out but the problem is this  apk can't generate module ...

**Comment by dedakraeveda** [comment]
> Did you tried without sai? I ment with official installer?
> 
> I wanted to give the module but every phone generate different files for module that's why it's not possible.
> But if you want you can follow this way,
> 
> 1. Download master from this repo & extract is. (it's a empty template)
> 2. Install official gltools.
> 3. generate magisk module
> 4. extract system folder from that module and place it in downloaded template.
> 5. zip it and install in magisk.
> 
> This should work.....

Hello! I did it according to this guide, Magisk work, but Gl Tools does not see the module. Can you help?
